server-side-rendering/src/routes/signin/+page.server.ts

```diff
@@ -0,0 +1,28 @@
+import { SESSION_COOKIE, createAppwriteClient } from '$lib/server/appwrite.js';
+import { redirect } from '@sveltejs/kit';
+
+export function load({ locals }) {
+	if (locals.user) throw redirect(301, '/');
+}
+
+export const actions = {
+	default: async (event) => {
```

Comment:
![CleanShot 2024-01-24 at 12 28 56@2x](https://github.com/appwrite/demos-for-svelte/assets/19310830/b788f6d2-9cac-41a2-a1f7-5f7adbdeba9a)

Hmm, any good way to proxy some of the exceptions to the client? for example email already used, wrong password, stuff like that

Reply:
Replaced with `error()` helper to show message